.travis.yml

```diff
@@ -1,12 +1,23 @@
 language: java
-sudo: false
+sudo: required
+dist: trusty
+# cache the dependency-check-maven plugin catalog (280MB+)
+cache:
+  directories: $HOME/.m2/repository/org/owasp/dependency-check-data/
 addons:
   apt:
     packages:
       - oracle-java8-installer
+      - google-chrome-stable
+    sources:
+      - google-chrome
 jdk:
   - oraclejdk8
 before_install:
+  - sudo apt-get install chromium-chromedriver
+before_script:
+  - "export PATH=$PATH:/usr/lib/chromium-browser/"
+  - "which chromedriver"
```

Comment:
We can remove this now, I mainly wanted this as a fall back for when the test might fail. and check if the driver plugin was properly installed